### PR TITLE
Revert "Revert "Add /breach-details recommendations""

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
Reverts mozilla/blurts-server#1440

### CODE REVIEW

File: public/js/fxa-analytics.js [hunk 1]
```diff
@@ -122,8 +122,17 @@ function getUTMNames() {
       },
     });
 
-    document.querySelectorAll(".send-ga-ping").forEach((el) => {
-      el.addEventListener("click", async(e) => {
+    // Temporarily send "View" and "Engage" pings for recommendation CTAs.
+    document.querySelectorAll(".recommendation-cta").forEach(cta => {
+      const eventLabel = cta.dataset.eventLabel;
+      ga("send", "event", "Breach Detail: Recommendation CTA", "View", eventLabel);
```
Comment (groovecoder) [inline public/js/fxa-analytics.js]: This has the same "bug" as https://github.com/mozilla/blurts-server/pull/1429/files?file-filters%5B%5D=.js#r357364009 ... i.e., it fires the "View" event even if the recommendation CTA is behind "See additional recommendations".

Can we fix that for a merge + deploy on Monday?

Comment (groovecoder) [reply public/js/fxa-analytics.js]: oooo, maybe a fancy `IntersectionObserver`?

Comment (lesleyjanenorton) [reply public/js/fxa-analytics.js]: I will do the thing!

Comment (groovecoder) [reply public/js/fxa-analytics.js]: Filed https://github.com/mozilla/blurts-server/issues/1445 for this.

**Pull request merged**